src/Radar.js

```diff
@@ -199,6 +214,16 @@ export default class Radar extends Viz {
     return this;
   }
 
+  /**
+      @memberof Radar
+      @desc If *value* is specified, sets the sum accessor to the specified function or number and returns the current class instance. If *value* is not specified, returns the current size accessor.
```

Comment:
I know I haven't been a stickler for this, but could you write a better description for this method? Right now all the descriptions sound like they were written by a robot 🤖